vcell-client/src/main/java/cbit/vcell/mapping/gui/MolecularStructuresPropertiesPanel.java

```diff
@@ -140,11 +166,39 @@ public boolean isViewSingleRow() {
                 }
             };
             shapePanel.addMouseMotionListener(new MouseMotionAdapter() {
+                @Override
                 public void mouseMoved(MouseEvent e) {
+                    // this will fire if the mouse moves over the panel, we may need it for drag and drop
+                    super.mouseMoved(e);
+//                    Point overWhat = e.getPoint();
```

Comment:
remove when you are done with this code.

Reply:
I'll probably remove the whole mouse motion listener eventually, unless it's needed for the drag and drop implementation